dependent_type_theory.md

```diff
@@ -55,14 +55,12 @@ in many programming languages.
 The ``def`` keyword declares new constant symbols into the
 working environment. In the example above, `def m : Nat := 1`
 defines a new constant `m` of type `Nat` whose value is `1`.
-The ``#check`` command asks Lean to report their
-types; in Lean, auxiliary commands that query the system for
-information typically begin with the hash (#) symbol.
-The `#eval` command asks Lean to evaluate the given expression.
-You should try
-declaring some constants and type checking some expressions on your
-own. Declaring new objects in this manner is a good way to experiment
-with the system.
+
+In Lean, auxiliary commands that query the system for information typically begin with the hash (#) symbol.
+The ``#check`` command asks Lean to report the type of a given expression
+types; the `#eval` command asks Lean to evaluate the given expression.
+Try declaring some constants and type checking some expressions on your
+own; this is a good way to experiment with the system.
```

Comment:
Successive sentences with semicolons is a bold choice - what about an em dash instead of the second one?